lib/turn.rb

```diff
@@ -1,6 +1,17 @@
 class Turn
-  def initialize()
+  attr_reader :board, :player1, :player2
+  def initialize(board, player1, player2)
+    @player1 = player1
+    @player2 = player2
+    @board = board
   end
 
-
+  def valid_input_length(move)
+    move.length == 1
+  end
+  
+  def valid_input_character(letter)
```

Comment:
Similar to above, can we add a '?' to the end of the method as the return value is boolean?